.github/workflows/test-erlang-git.yaml

```diff
@@ -0,0 +1,45 @@
+name: Test Erlang Git Master
+on:
+  schedule:
+  - cron: '0 2 * * *'
+  workflow_dispatch:
+  push:
+jobs:
+  test-erlang-git:
+    name: Test (Erlang Git Master)
+    runs-on: ubuntu-18.04
+    timeout-minutes: 120
+    steps:
+    - name: CHECKOUT REPOSITORY
+      uses: actions/checkout@v2.3.4
+    - name: UPDATE RBE IMAGE SHA
+      run: |
+        # buildbuddy caches the container image, so we must use a specific sha to ensure
+        # the latest is used
+        IMAGE=pivotalrabbitmq/rabbitmq-server-buildenv:linux-erlang-git-master
+        docker pull ${IMAGE}
```

Comment:
Is it only fetched to get sha256? I was able to get the same value using 
```
skopeo inspect docker://pivotalrabbitmq/rabbitmq-server-buildenv:linux-erlang-git-master | jq .Digest
``

As `docker pull` fetches close to 1Gb of data, and it's over 3Gb unpacked.

Reply:
Yes, it's only fetched for the sha. In my googling I didn't find the `skopeo` approach, but I would say it's a good improvement. This is already merged so feel free to update it in place or a new PR, whichever you prefer.

Reply:
#3218